axelrod/strategies/finite_state_machines.py

```diff
@@ -1024,3 +1024,71 @@ def __init__(self) -> None:
         super().__init__(
             transitions=transitions, initial_state=0, initial_action=C
         )
+
+ class EvolvedFSM8(FSMPlayer):
+    """
+    An 8 state FSM player trained with an evolutionary algorithm.
+    
+    Evolved from the 10 state FSM designed by Frederick Vincent with transitions
+    
+    transitions = (
+            (1, C, 2, C),
```

Comment:
Note that this comment is mostly copied from the 10 state strategy and isn't updated for the transitions below.

Reply:
We started with the 10 state FSM, but after 500 generations the output was an 8 state given in the class below. You are right, I did copy the comment for a consistent look.